docs/source/community/index.md

```diff
@@ -1,7 +1,8 @@
 # Community
 
-Contributions to `movement` are absolutely encouraged, whether to fix a bug,
-develop a new feature, or improve the documentation.
+`movement` is a community-driven project, made possible by the generous contributions of [many people](target-people).
```

Comment:
```suggestion
`movement` is made possible by the generous contributions of [many people](target-people).
```

I think it's not 100% genuine to say we are 'community-driven' in the strict sense. We definitely invite and welcome community feedback and contributions, but the "drive" comes from the NIU (the general mission and scope have been set by us and this will remain the case).